mcs/class/Mono.Profiler.Log/Makefile

```diff
@@ -2,14 +2,26 @@ thisdir = class/Mono.Profiler.Log
 include ../../build/rules.make
 
 LIBRARY = Mono.Profiler.Log.dll
-LIBRARY_SNK = ../mono.snk
+LIBRARY_WARN_AS_ERROR = yes
 
-LIB_REFS = System System.Core
 KEYFILE = $(LIBRARY_SNK)
+
+LIB_REFS = System System.Core
 LIB_MCS_FLAGS = /unsafe /publicsign /nowarn:0618
 
-LIBRARY_WARN_AS_ERROR = yes
+XTEST_LIB_REFS = System System.Core
+
+xunit-test-local: log-profiler-test.exe
```

Comment:
We already have support for this, called REMOTE_EXECUTOR see https://github.com/mono/mono/blob/master/mcs/class/System/Makefile#L31 how to enable it. 

You then derive from RemoteExecutorTestBase and write your asserts as normal code

```
    public class Foo : RemoteExecutorTestBase
    {
        [Fact]
        public static void Test()
        {
            RemoteInvoke(() => { });
        }
```

I think it'd make sense to use this instead of building another one

Reply:
I don't think that would apply to this use case since I need to pass arguments to the runtime itself?

Reply:
Then just extend it, it's still better than maintaining two very similar support code